src/lib/include/libsed.h

```diff
@@ -70,6 +70,11 @@ int sed_lock_unlock(struct sed_device *dev, const struct sed_key *key, enum SED_
  */
 int sed_reverttper(struct sed_device *dev, const struct sed_key *key, bool psid);
 
+/**
+ *
+ */
+int sed_revertsp(struct sed_device *dev, const struct sed_key *key, bool keep_global_rn_key);
```

Comment:
Suggest renaming function to sed_revertlsp to indicate that revert is happening on Locking SP.

Reply:
done